code/modules/mob/new_player/preferences_setup.dm

```diff
@@ -207,751 +207,73 @@
-				switch(backbag)
-					if(2)
-						clothes_s.Blend(new /icon('icons/mob/back.dmi', "backpack"), ICON_OVERLAY)
-					if(3)
-						clothes_s.Blend(new /icon('icons/mob/back.dmi', "satchel-eng"), ICON_OVERLAY)
-					if(4)
-						clothes_s.Blend(new /icon('icons/mob/back.dmi', "satchel"), ICON_OVERLAY)
-			if(LAWYER)
-				clothes_s = new /icon(uniform_dmi, "internalaffairs_s")
-				clothes_s.Blend(new /icon('icons/mob/feet.dmi', "brown"), ICON_UNDERLAY)
-				clothes_s.Blend(new /icon('icons/mob/inhands/items_righthand.dmi', "briefcase"), ICON_UNDERLAY)
-				if(prob(1))
-					clothes_s.Blend(new /icon('icons/mob/suit.dmi', "suitjacket_blue"), ICON_OVERLAY)
-				switch(backbag)
-					if(2)
-						clothes_s.Blend(new /icon('icons/mob/back.dmi', "backpack"), ICON_OVERLAY)
-					if(3)
-						clothes_s.Blend(new /icon('icons/mob/back.dmi', "satchel-norm"), ICON_OVERLAY)
-					if(4)
-						clothes_s.Blend(new /icon('icons/mob/back.dmi', "satchel"), ICON_OVERLAY)
-			if(CHAPLAIN)
-				clothes_s = new /icon(uniform_dmi, "chapblack_s")
-				clothes_s.Blend(new /icon('icons/mob/feet.dmi', "black"), ICON_UNDERLAY)
-				if(prob(1))
-					clothes_s.Blend(new /icon('icons/mob/suit.dmi', "imperium_monk"), ICON_OVERLAY)
-				switch(backbag)
-					if(2)
-						clothes_s.Blend(new /icon('icons/mob/back.dmi', "backpack"), ICON_OVERLAY)
-					if(3)
-						clothes_s.Blend(new /icon('icons/mob/back.dmi', "satchel-norm"), ICON_OVERLAY)
-					if(4)
-						clothes_s.Blend(new /icon('icons/mob/back.dmi', "satchel"), ICON_OVERLAY)
-			if(CLOWN)
-				clothes_s = new /icon(uniform_dmi, "clown_s")
-				clothes_s.Blend(new /icon('icons/mob/feet.dmi', "clown"), ICON_UNDERLAY)
-				clothes_s.Blend(new /icon('icons/mob/mask.dmi', "clown"), ICON_OVERLAY)
-				clothes_s.Blend(new /icon('icons/mob/back.dmi', "clownpack"), ICON_OVERLAY)
-			if(MIME)
-				clothes_s = new /icon(uniform_dmi, "mime_s")
-				clothes_s.Blend(new /icon('icons/mob/feet.dmi', "black"), ICON_UNDERLAY)
-				clothes_s.Blend(new /icon('icons/mob/hands.dmi', "lgloves"), ICON_UNDERLAY)
-				clothes_s.Blend(new /icon('icons/mob/mask.dmi', "mime"), ICON_OVERLAY)
-				clothes_s.Blend(new /icon('icons/mob/head.dmi', "beret"), ICON_OVERLAY)
-				clothes_s.Blend(new /icon('icons/mob/suit.dmi', "suspenders"), ICON_OVERLAY)
-				switch(backbag)
-					if(2)
-						clothes_s.Blend(new /icon('icons/mob/back.dmi', "backpack"), ICON_OVERLAY)
-					if(3)
-						clothes_s.Blend(new /icon('icons/mob/back.dmi', "satchel-norm"), ICON_OVERLAY)
-					if(4)
-						clothes_s.Blend(new /icon('icons/mob/back.dmi', "satchel"), ICON_OVERLAY)
-
-	else if(job_medsci_high)
-		switch(job_medsci_high)
-			if(RD)
-				clothes_s = new /icon(uniform_dmi, "director_s")
-				clothes_s.Blend(new /icon('icons/mob/feet.dmi', "brown"), ICON_UNDERLAY)
-				clothes_s.Blend(new /icon('icons/mob/suit.dmi', "labcoat_open"), ICON_OVERLAY)
-				if(prob(1))
-					clothes_s.Blend(new /icon('icons/mob/head.dmi', "petehat"), ICON_OVERLAY)

```

Comment:
these little rare things are the only thing I'll miss.

I once had a metroid on my head as a scientist---a bear belt, as a shaft miner, another time.

Reply:
Yeah when I saw that shit I thought it was a bug and had a good laugh. RIP!

Reply:
Anyway we could add it back somehow?

Reply:
Could probably factor in 'chance to assign fluff kit' into outfitting, with a list of defined object datums and their chances for use in `prob()`

Reply:
For now, I wouldn't worry about it; let's get this system, as is, in and functioning, since it's gone long enough without being merged and is already giving us enough trouble.

That said, it being re-added in the future, if it can be done in a non-snowflakey way (that'll be hard) would be great.

Reply:
agreed. just got to figure out the other stuff. But come as is...Hopeffully this big wet windstorm off the coast dosen't give me a time.